src/PackageManager/NodePackageManager.php

```diff
@@ -97,4 +97,28 @@ public function getName(): string
     {
         return 'Node';
     }
+
+    /**
+     * "npm list" for npm >= 8 does not return the description field in a reliable way.
+     * It may be present or it may be missing. In case "npm list" doesn't return it, we'll
+     * look into the package's own package.json, parse it and try to fetch the description
+     * from there.
+     */
+    private function determinePackageDescription(string $packagePath): ?string
```

Comment:
Smart workaround!
I'm excited to see this working 🙌🏼

Reply:
So am I, great stuff, thanks a lot! I'm a bit worried about the run time, considering the huge pile of dependencies we have in our project, but we will see :)

I' gonna merge this now (and also #34) and create a new major release after that! Thanks again <3